kimchi/benches/proof_criterion.rs

```diff
@@ -5,26 +5,30 @@ pub fn bench_proof_creation(c: &mut Criterion) {
     let mut group = c.benchmark_group("Proof creation");
     group.sample_size(10).sampling_mode(SamplingMode::Flat); // for slow benchmarks
 
-    let ctx = BenchmarkCtx::new(10);
-    group.bench_function(
-        format!("proof creation (SRS size 2^{})", ctx.srs_size()),
-        |b| b.iter(|| black_box(ctx.create_proof())),
-    );
+    for size in [10, 14] {
+        let ctx = BenchmarkCtx::new(size);
 
-    let ctx = BenchmarkCtx::new(14);
-    group.bench_function(
-        format!("proof creation (SRS size 2^{})", ctx.srs_size()),
-        |b| b.iter(|| black_box(ctx.create_proof())),
-    );
-
-    let proof_and_public = ctx.create_proof();
+        group.bench_function(
+            format!("proof creation ({} gates)", ctx.num_gates),
```

Comment:
Can you add the SRS size, in log2 please? We often think in terms of log2 domain size for the benchmarks. It is good to have it on the stdout.

Reply:
Sure!

(The reason I had added the number of gates was that the size in log2 alone was not leading to unique names, but having both solves the issue and keeps the log2 size).